lib/client.js

```diff
@@ -349,12 +349,25 @@ Client.prototype.loadMetadataForTopics = function (topics, cb) {
 };
 
 Client.prototype.createTopics = function (topics, isAsync, cb) {
+  function isValid (str) {
+    return !/[~`!#$%\^&*+=\-\[\]\\';,/{}|\\":<>\?]/g.test(str);
+  }
   topics = typeof topics === 'string' ? [topics] : topics;
 
   if (typeof isAsync === 'function' && typeof cb === 'undefined') {
     cb = isAsync;
     isAsync = true;
   }
+  var validTopics = true;
+  Object.keys(topics).map(function (elem) {
```

Comment:
`Array.some` could be used here and may not need to iterate through the entire list.